tools/harness-simulation/README.md

```diff
@@ -10,15 +10,32 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 
 ## POSIX Environment Setup
 
-1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, run the following command in the top directory of OpenThread.
+1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. Taking running OpenThread 1.2 test cases as an example, run the following command in the top directory of OpenThread.
 
    ```bash
-   $ script/cmake-build simulation
+   $ CFLAGS='-DOPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS=8' \
+     CXXFLAGS='-DOPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS=8' \
+     script/cmake-build simulation \
+         -DOT_THREAD_VERSION=1.2 \
+         -DOT_DUA=ON \
+         -DOT_MLR=ON \
+         -DOT_COMMISSIONER=ON \
+         -DOT_CSL_RECEIVER=ON
    ```
 
    Then `ot-cli-ftd` is built in the directory `build/simulation/examples/apps/cli/`.
 
-2. Run the installation script.
+2. Check the configuration file `config.py`
+
+   - Edit the value of `OT_PATH` to the absolute path where the top directory of the OpenThread repository is located.
+
+3. Run the `build_docker_image.sh` with the environment variable `OT_PATH` set properly. For example run the following command.
+
+   ```bash
+   $ OT_PATH=~/repo/openthread ./build_docker_image.sh
```

Comment:
Why is `OT_PATH` required when we are already in the openthread directory?

Reply:
The directory of the scripts and the related configuration may alter in the following PR.